pcsx2/gui/Panels/MemoryCardListPanel.cpp

```diff
@@ -629,12 +630,13 @@ void Panels::MemoryCardListPanel_Simple::UiCreateNewCard( McdSlotItem& card )
 
 	if (result != wxID_CANCEL) {
 		card.IsEnabled = true;
-		card.Filename  = dialog.result_createdMcdFilename;
+		card.Filename  = dialog.result_createdMcdFilename.ToStdString();
 		card.IsPresent = true;
+		wxString toPrint(card.Filename.wstring());
 		if (card.Slot >= 0) {
-			Console.WriteLn(L"Setting new memory card to slot %u: '%s'", card.Slot, WX_STR(card.Filename.GetFullName()));
+			Console.WriteLn(L"Setting new memory card to slot %u: '%s'", card.Slot, WX_STR(toPrint));
 		} else {
-			Console.WriteLn(L"Created a new unassigned memory card file: '%s'", WX_STR(card.Filename.GetFullName()));
+			Console.WriteLn(L"Created a new unassigned memory card file: '%s'", WX_STR(toPrint));
```

Comment:
similar comment to the one above, i assume maybe this is required because of macro shenanigans?

Reply:
Ye